### PR TITLE
Add WriterT Bifunctor

### PR DESCRIPTION
**Pull request by ceedubs**
Knocks one item off of the checklist on #620.

### CODE REVIEW

File: core/src/main/scala/cats/data/WriterT.scala [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package cats
 package data
 
+import functor.Bifunctor
```
Comment (adelbertc) [inline core/src/main/scala/cats/data/WriterT.scala]: Nitpicking, but given we seem to use the fully qualified name in other imports, maybe `import cats.functor.Bifunctor`

**Comment by adelbertc** [comment]
:+1:

File: core/src/main/scala/cats/data/WriterT.scala (hunk 1 above)
Comment (ceedubs) [inline core/src/main/scala/cats/data/WriterT.scala]: I like your style. Will do.

**Comment by fthomas** [comment]
:+1:

**Pull request merged**